pyls/plugins/pycodestyle_lint.py

```diff
@@ -1,8 +1,13 @@
 # Copyright 2017 Palantir Technologies, Inc.
 import logging
 import pycodestyle
+from autopep8 import continued_indentation as autopep8_c_i
 from pyls import hookimpl, lsp
 
+if autopep8_c_i in pycodestyle._checks['logical_line']:
```

Comment:
Please add a comment above this line to explain the reasoning behind this block.

Reply:
Gladly 👍